pkg/grapicmd/di/providers.go

```diff
@@ -70,23 +77,37 @@ func ProvideGexConfig(ctx *grapicmd.Ctx) *gex.Config {
 	return gexCfg
 }
 
-func ProvideInitializeProjectUsecase(ctx *grapicmd.Ctx, gexCfg *gex.Config, ui clui.UI, generator module.Generator) usecase.InitializeProjectUsecase {
-	return usecase.NewInitializeProjectUsecase(
-		ui,
-		generator,
-		gexCfg,
-		ctx.Version,
-	)
+func ProvideToolRepository(ctx *grapicmd.Ctx, gexCfg *gex.Config) (tool.Repository, error) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ProvideToolRepository should have comment or be unexported